creategameprojects_dev.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 chmod u+x ./devtools/bin/vpc
-devtools/bin/vpc /define:WORKSHOP_IMPORT_DISABLE /define:SIXENSE_DISABLE /define:NO_X360_XDK /define:RAD_TELEMETRY_DISABLED /define:DISABLE_ETW /define:LTCG /no_ceg /retail /define:CERT /nofpo /tf +game /mksln games
```

Comment:
Why?

Reply:
The creategameprojects_dev.sh is to make TC2 with flags that enable better dev options.

Reply:
But creategameprojects_dev.bat has it..

Reply:
Yes, because it is an optimization that shouldn't affect debugging too much. Debug versions are supposed to have the least amount of optimizations as possible.